src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -95,11 +96,9 @@ func (client *MobileClient) recordAnalytics() {
 		},
 	}
```

Comment:
I initially removed creating the session payload here since a lot of this code is identical to the code found in ``flashlight/analytics.go``, but that package unfortunately imports ``config`` and referencing it at all has caused various issues in the past. I'm inclined to duplicate the code for now until we have time to properly address the larger issue.